zokrates_book/src/sha256example.md

```diff
@@ -13,7 +13,7 @@ Make sure you have followed the instructions in the [Getting Started](gettingsta
 
 We will start this tutorial by using ZoKrates to compute the hash for an arbitrarily chosen preimage, being the number `5` in this example.
 
-First, we create a new file named `hashexample.code` with the following content:
+First, make sure that the `zokrates` binary is located in the `ZoKrates` directory. Then, we create a new file named `hashexample.code` with the following content:
```

Comment:
Why is that? If it's needed, do we have an easy way to make that more seamless?

Reply:
not needed any more. this was still based on relative imports when we didn't have support for absolute imports